### ISSUE TITLE
IEconItems_440/GetSchema is deprecated

### ISSUE THREAD

**Issue by aleksamagicka** [babelshift/SteamWebAPI2#66]
IEconItems_440/GetSchema is now deprecated (returns HTTP 410). Here is what it returns:

```
{
	"result": {
		"status": 80,
		"note": "Deprecated: See GetSchemaOverview and GetSchemaItems"
	}
}
```

More info [here](https://www.reddit.com/r/tf2/comments/8glw2m/website_operators_and_update_enthusiasts_that/) and [here](https://twitter.com/wickedplayer494/status/991822413066219520).

**Comment by CriticalFlaw** [comment]
Is there an update on when/if this will be fixed?

**Comment by babelshift** [comment]
Fixed with #82
